src/js/index.js

```diff
@@ -14,7 +14,8 @@ const sort = {
 const contentEl = document.getElementById('content');
 const filterInput = document.getElementById('filter');
 const sortingInput = document.getElementById('sorting');
-
+const color_switch = document.getElementById('color-switch');
```

Comment:
Please use `lowerCamelCase` for your variable names (`color_switch` becomes `colorSwitch`)